### DISCUSSION TITLE
Suddenly no redirect to payment gate

### DISCUSSION THREAD

**Discussion by JanineKall** [liedekef/events-made-easy#699] [General]
Happy Christmas to begin with!
In our website https://hb-cafe.nl we've used EME for ages. We use Mollie as our payment gateway and it always worked fine. 
Since a few days, some customers indicate they are not redirected to the payment gateway, especially not for Flevoland and Zwolle-events, but there could be more events involved. We do receive some payments on other events on the same website though. 
I don't know why. As is it Christmas, it's not as busy as it usually is. 
To my knowledge, no setting has been changed and I can find no errors in the settings. Do you know what's happening, if there's something happening at all? No rush, as it is Christmas ;-)

**Comment by liedekef** [comment]
I tried subscribing to a Zwolle-event (you can delete it), but no redirect happens. Some possibilities: the price is 0 or no payment method is selected.

**Reply by liedekef** [reply]
btw: I still seem to have an account, I'll check in the backend

**Reply by liedekef** [reply]
ok, found the issue: typo in the code. But I made other changes too, so I'll verify the code and release an update.

**Reply by liedekef** [reply]
all fixed (new release done too and I fixed your 2 events: due to a typo there was no payment gateway configured, next time check the payment methods in the RSVP section of the event, there the problem will show :-) )

**Reply by JanineKall** [reply]
Oh thank you so much! I saw there was no payment gateway in de RSVP section, but I did not know why, as it was configured in the settings! What typo did I do?

**Reply by liedekef** [reply]
it was a typo I made in the code :-)

**Reply by JanineKall** [reply]
It's a relief that we are all human after all. Have a nice tweede kerstdag!

**Reply by JanineKall** [reply]
P.S. I can't find your subscription to a Zwolle event, not even in recycle bin?

**Reply by liedekef** [reply]
I removed it, after I fixed things on your end. Please also deactivate my account :-)

**Reply by liedekef** [reply]
Btw, thank you for the nice donation!
